src/OrchardCore.Modules/OrchardCore.Email/Services/SmtpService.cs

```diff
@@ -132,16 +132,19 @@ private bool CertificateValidationCallback(object sender, X509Certificate certif
 
             if (!valid)
             {
-                _logger.LogError($"SMTP Server's certificate {certificate.Subject} issued by {certificate.Issuer} " +
-                    $"with thumbprint {certificate.GetCertHashString()} and expiration date {certificate.GetExpirationDateString()} " +
-                    $"is considered invalid with {sslPolicyErrors} policy errors");
+                _logger.LogError(string.Concat("SMTP Server's certificate {CertificateSubject} issued by {CertificateIssuer} ",
+                    "with thumbprint {CertificateThumbprint} and expiration date {CertificateExpirationDate} ",
+                    "is considered invalid with {SslPolicyErrors} policy errors"),
+                    certificate.Subject, certificate.Issuer, certificate.GetCertHashString(),
+                    certificate.GetExpirationDateString(), sslPolicyErrors);
             }
 
             if (sslPolicyErrors.HasFlag(SslPolicyErrors.RemoteCertificateChainErrors))
             {
                 for (int i = 0; i < chain.ChainStatus.Length - 1; i++)
                 {
-                    _logger.LogError($"{i} = {chain.ChainStatus[i].StatusInformation} : {chain.ChainStatus[i].Status}");
+                    _logger.LogError("chainStatus[{ChainStatusIndexi}] = {Status}:{StatusInformation} ",
```

Comment:
I don't see the value of including the index. Just having them all outputed should be fine. Using a custom column for the index feels dirty to me.